0000-template.md

```diff
@@ -4,7 +4,7 @@
 * Proposal: [SDL-NNNN](NNNN-filename.md)
 * Author: [SDL Developer](https://github.com/smartdevicelink)
 * Status: **Awaiting review**
-* Impacted Platforms: [Core / iOS / Android / Web / RPC / Protocol]
+* Impacted Platforms: [Core / iOS / Android / Cloud / HMI / Policy Server / SHAID / RPC / Protocol]
```

Comment:
Perhaps also add `Web Proxy` to indicate the JavaEE proxy library?

Reply:
I've updated to `Cloud Proxy` per our conversation